### DISCUSSION TITLE
keycloak login invalid request

### DISCUSSION THREAD

**Discussion by angellaviory** [keycloak/keycloak#22682] [Q&A]
hello, i'm still new in keycloak. I already make client (either with IP:port or domain), but when i login i got this message:

[org.keycloak.events] (executor-thread-19) type=LOGIN_ERROR, realmId=a9706c94-47d4-431a-ad57-11fa19c05747, clientId=null, userId=null, ipAddress=[---], error=invalid_request

can someone help?

**Comment by yanikiss** [comment]
Hello! Did you manage to solve the problem? I have the same
